src/strava_py/plot_map.py

```diff
@@ -0,0 +1,34 @@
+import matplotlib.pyplot as plt
+
+def plot_map(df, lon_min=None, lon_max= None, lat_min=None, lat_max=None,
+             alpha=0.3, linewidth=0.3, output_file="map.png"):
+    
```

Comment:
Turns out we need to explicitly create a new figure here and same place in `plot_facets`:

```suggestion
    
    # Create a new figure
    plt.figure()
```

Otherwise the map plot is like this -- a facet plot with the map in the last slot:

![image](https://user-images.githubusercontent.com/1324225/152685843-58992765-c453-4932-840e-6912a7e91513.png)